docker/release.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Usage:
+# // To build and publish the Docker image to the internal container registry.
+# $ PUBLIC_DESTINATION=internal bash release.sh
```

Comment:
This should be triggered automatically by the container image trigger.

Reply:
Why `PUBLIC_DESTINATION` and not `DESTINATION`?

Reply:
How about using a positional parameter instead of the environment variable? E.g.,
```
bash release.sh build
```
and
```
bash release.sh publish
```
?

Reply:
Environment variables are good for wrapper scripts like `entrypoint.sh`, where positional parameters would interfere with the parameters of the wrapped executable. Another good use case for environment variables is overriding default behavior, like we do in our install scripts. For build scripts like this one, it's more natural to say `./release.sh build` than `DESTINATION=internal ./release.sh` (IMO).

Reply:
I meant the Kokoro scripts that we own. They are all using environment variables. This script will be used by the Kokoro infra.

Reply:
Ok, then why are we committing this to the open-source repo, rather than keeping it in the Kokoro scripts directory?